src/public/video.ts

```diff
@@ -99,13 +131,15 @@ export namespace video {
    * in-meeting scenario, we will call videoEffectCallback when apply button clicked.
    * @param effectChangeType the effect change type.
    * @param effectId Newly selected effect id.
+   * @param effectParam param for the selected effect.
    */
   export function notifySelectedVideoEffectChanged(
     effectChangeType: EffectChangeType,
     effectId: string | undefined,
+    effectParam: string | undefined,
```

Comment:
Shall we make the effectParam to be optional? Since for videoApp not used this parameter.

Reply:
Changed to optional